### PR TITLE
Update Appendix V to include how SPDX-FileCopyrightText

### PR DESCRIPTION
**Pull request by kestewart**
This has been agreed between the SPDX and REUSE community as the syntax to enable Copyright Notices to be better detected by scanning tools.    It was published by REUSE as part of specification version 3.0 (https://github.com/fsfe/reuse-docs/blob/master/spec.md#format-of-copyright-notices), and discussed in SPDX tech team calls and issue https://github.com/spdx/spdx-spec/issues/122

### CODE REVIEW

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md [hunk 1]
```diff
@@ -52,4 +52,22 @@ Examples:
...
+
+SPDX-FileCopyrightNotice: <Copyright Notice Information>
+
+The copyright notice must have a copyright holder identified, which may be an individual, list of individuals, group, legal entity, or any other descriptor by which one can easily identify the copyright holder(s).    It may also contain a single year, or range of years.
+
+Examples of valid copyright notices:
+
+   SPDX-FileCopyrightText: 2019 Jane Doe <jane@example.com>
+   SPDX-FileCopyrightText: 2019 Example Company
+   SPDX-FileCopyrightText: The Authors of
```
[the remark this answers is not in the document]
Comment (kestewart) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Thanks Carmen.  Good catch.  Multiple drafts merged, my bad. Fixing.

[the remark this answers is not in the document]
Comment (kestewart) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Please re-review.   Thanks!

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md [hunk 2]
```diff
@@ -52,4 +52,22 @@ Examples:
...
+
+The SPDX-FileCopyrightText tag declares the copyright that the file is under and should be placed at or near the top of the file in a comment. To the extent that the file contains existing copyright notices, it is our recommendation that the tag be used to supplement not replace that information. Existing Copyright notices should *not* be removed from a file without the consent of the copyright holder. 
+
+The tag should appear on its own line in the source file, generally as part of a comment.
+
+SPDX-FileCopyrightText: \<Copyright Text Information\>
+
+The Copyright Text Information must have a copyright holder identified, which may be an individual, list of individuals, group, legal entity, or any other descriptor by which one can easily identify the copyright holder(s).    It may also contain a single year, or range of years.
```
Comment (swinslow) [inline chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Please see my overall comments below, but in particular I would remove this line. To my knowledge SPDX does not make recommendations about the particular content of copyright notices.

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md [hunk 3]
```diff
@@ -52,4 +52,22 @@ Examples:
...
+
+Examples of valid copyright notices:
+
+    SPDX-FileCopyrightText: 2019 Jane Doe <jane@example.com>
+    SPDX-FileCopyrightText: 2019 Example Company
+    SPDX-FileCopyrightText: The Authors of
```
Comment (swinslow) [inline chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: This last line appears to have been cut off...

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md [hunk 4]
```diff
@@ -52,4 +52,22 @@ Examples:
...
\ No newline at end of file
+If you can’t express the license(s) as an expression using identifiers from the SPDX list, it is probably best to just put the text of your license header in the file (if there is a standard header), or refer to a neutral site URL where the text can be found. To request a license be added to the SPDX License List, please follow the process described here: [http://spdx.org/spdx-license-list/request-new-license-or-exception](http://spdx.org/spdx-license-list/request-new-license-or-exception).
+
+## Format for SPDX-FileCopyrightText
+
+Some licenses require the identification of the copyright holders.   The natural language syntax can result in many false positives being picked up by source code scanning tools.  To be able to denote the copyright that tools should pick up for a file, the [REUSE project](https://reuse.software/spec/) and SPDX community standardized on the identifier of SPDX-FileCopyrightText.
```
Comment (swinslow) [inline chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Please see my overall comments below, but in particular I would remove the first sentence, "Some licenses require..."

**Comment by swinslow** [comment]
@kestewart I'm putting together a proposed alternative appendix to tackle this in the way I mentioned above -- will circle back shortly with it for your and others' review.

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (carmenbianca) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: The tag has been correctly renamed. The last line is still incomplete/cut off, as also remarked by @swinslow

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md (hunk 2 above)
Comment (mxmehl) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: For the sake of completeness, here's REUSE's take on this: https://reuse.software/faq/#standard-copyright

What would be the use of SPDX-FileCopyrightText if there is no copyright holder defined?

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md [hunk 5]
```diff
@@ -24,9 +24,9 @@ The SPDX License Identifier syntax may consist of a single license (represented
 
 The tag should appear on its own line in the source file, generally as part of a comment.
 
-SPDX-License-Identifier: <SPDX License Expression>
+SPDX-License-Identifier: \<SPDX License Expression\>
 
-## Representing Single License
+### Representing Single License
 
 A single license is represented by using the short identifier from [SPDX license list](https://spdx.org/licenses/), optionally with a unary "+" operator following it to indicate "or later" versions may be applicable.
```
Comment (mxmehl) [inline chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: I accidentally read this. Aren't license identifiers with `+` deprecated, at least the well-known GPLs?

Comment (carmenbianca) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: As far as I understand, `+` is deprecated *only* for GPL, and not for any other licenses. `CC-BY-SA-3.0+` is valid SPDX. Correct me if I'm wrong.

Comment (kestewart) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: "+" is valid syntax on other licenses.    I spotted an old style GPL reference that was in the prior version, and went in and updated it.

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md (hunk 3 above)
Comment (kestewart) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Fixed now.    Thanks for spotting.

File: chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md (hunk 4 above)
Comment (kestewart) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Ok.   Standing by for an alternative to apply.    Thanks!

Comment (swinslow) [reply chapters/appendix-V-using-SPDX-short-identifiers-in-source-files.md]: Hi @kestewart, I proposed an alternative PR at #196 -- take a look and see what you think.

**Comment by kestewart** [comment]
Closing request in favor of using #196

**Pull request closed without merging**